<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,20,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="toolsLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="selectButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Select object</string>
          </property>
          <property name="text">
           <string>Sel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="moveButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Move selected objects (G)</string>
          </property>
          <property name="text">
           <string>Mov</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="rotateButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Rotate selected objects (R)</string>
          </property>
          <property name="text">
           <string>Rot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="scaleButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Transform selected objects (T)</string>
          </property>
          <property name="text">
           <string>Sca</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tileBrushButton">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Bru</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="workSpaceLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="outlinerLayout" stretch="0,1">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="CTreeView" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="uniformRowHeights">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerMinimumSectionSize">
           <number>35</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mouseCoordLabel">
        <property name="text">
         <string>Mouse Coords:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mousePosText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusIco">
        <property name="text">
         <string>ICO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="statusBar"/>
      </item>
      <item>
       <widget class="CProgressView" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_recent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_all_MOB_s"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_landscape_MPR"/>
    <addaction name="actionSave_landscape_MPR_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose_all"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuSelect_2">
     <property name="title">
      <string>Select</string>
     </property>
     <addaction name="actionSelect_by"/>
     <addaction name="actionSelect_All"/>
    </widget>
    <addaction name="actionShow_undo_redo"/>
    <addaction name="actionCopy_IDs_to_clipboard"/>
    <addaction name="actionReset_cam_position"/>
    <addaction name="actionCreate_new_object"/>
    <addaction name="menuSelect_2"/>
    <addaction name="actionRandomize_parameter"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuUnit">
     <property name="title">
      <string>&amp;Unit</string>
     </property>
     <addaction name="actionReset_logic_paths"/>
    </widget>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
    <addaction name="action_Mob_parameters"/>
    <addaction name="actionChange_mod_e"/>
    <addaction name="menuUnit"/>
    <addaction name="actionReset_selected_IDs"/>
    <addaction name="separator"/>
    <addaction name="actionMap_parameters"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpen_2"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save active MOB</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save active MOB as...</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="actionSelect_by">
   <property name="text">
    <string>Select by...</string>
   </property>
  </action>
  <action name="actionCreate_new_object">
   <property name="text">
    <string>Create &amp;new object</string>
   </property>
   <property name="toolTip">
    <string>Create new object</string>
   </property>
  </action>
  <action name="actionShow_undo_redo">
   <property name="text">
    <string>Show &amp;history</string>
   </property>
  </action>
  <action name="action_Mob_parameters">
   <property name="text">
    <string>&amp;Mob parameters</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Un&amp;do</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSave_all_MOB_s">
   <property name="text">
    <string>Save all MOB's</string>
   </property>
  </action>
  <action name="actionReset_cam_position">
   <property name="text">
    <string>Reset &amp;cam position</string>
   </property>
  </action>
  <action name="actionChange_mod_e">
   <property name="text">
    <string>Change mod&amp;e</string>
   </property>
  </action>
  <action name="actionCopy_IDs_to_clipboard">
   <property name="text">
    <string>Copy &amp;IDs to clipboard</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionOpen_recent">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open &amp;recent</string>
   </property>
  </action>
  <action name="actionReset_logic_paths">
   <property name="text">
    <string>Reset &amp;logic paths</string>
   </property>
  </action>
  <action name="actionRandomize_parameter">
   <property name="text">
    <string>Randomize parameter</string>
   </property>
  </action>
  <action name="actionReset_selected_IDs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Reset selected IDs</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSave_landscape_MPR">
   <property name="text">
    <string>Save landscape (MPR)</string>
   </property>
  </action>
  <action name="actionSave_landscape_MPR_as">
   <property name="text">
    <string>Save landscape (MPR) as...</string>
   </property>
  </action>
  <action name="actionMap_parameters">
   <property name="text">
    <string>Map parameters</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CProgressView</class>
   <extends>QProgressBar</extends>
   <header>layout_components/progressview.h</header>
  </customwidget>
  <customwidget>
   <class>CTreeView</class>
   <extends>QTreeWidget</extends>
   <header>layout_components/tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
